readme.txt

```diff
@@ -118,7 +118,12 @@ Currently, the following add-ons are available for Event Tickets:
 == Changelog ==
 
 = [TBD] TBD =
+<<<<<<< HEAD
+
+* Fix - Correctly get the event id within the `rsvp_process` method in `/src/Tribe/Editor/Blocks/Rsvp.php` [135409]
```

Comment:
```suggestion
* Tweak - Add a check for empty tickets to `ticket_availability()` method to avoid notices. [122334]
* Fix - Correctly get the event id within the `rsvp_process` method in `/src/Tribe/Editor/Blocks/Rsvp.php` [135409]
```